examples-gallery/plot_two_link_pendulum_on_a_cart.py

```diff
@@ -0,0 +1,276 @@
+# %%
+"""
+=========================
```

Comment:
I realize I told you this wrong. I think it does not need the upper set of `====`. It wasn't making the thumbnail and then I removed it and it worked.

Reply:
corrected.